<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>nazwa pitcy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_size">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>srednica pitcy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_price">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>cena pitcy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>picca</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="combo2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pizza1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pizza2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_porownaj">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Por√≥wnaj</string>
   </property>
  </widget>
  <widget class="QLabel" name="porownanie">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>porownanie</zorder>
  <zorder>btn_add</zorder>
  <zorder>edit_name</zorder>
  <zorder>edit_size</zorder>
  <zorder>edit_price</zorder>
  <zorder>combo1</zorder>
  <zorder>combo2</zorder>
  <zorder>pizza1</zorder>
  <zorder>pizza2</zorder>
  <zorder>btn_porownaj</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
